src/DssSpell.t.sol

```diff
@@ -1004,128 +1071,53 @@ contract DssSpellTest is DssSpellTestBase {
-        require(L2BridgeSpellLike(L2_SPELL).l2Bridge() == address(l2Bridge),   "L2Spell/l2-bridge-mismatch");
-
-        rootDomain.selectFork();
-
-        require(keccak256(bytes(l1Bridge.version()))   == keccak256("1"),      "BaseTokenBridge/version-does-not-match");
-        require(l1Bridge.getImplementation()           == L1_BRIDGE_IMP,       "BaseTokenBridge/imp-does-not-match");
-        require(l1Bridge.isOpen()                      == 1,                   "BaseTokenBridge/not-open");
-        require(l1Bridge.otherBridge()                 == address(l2Bridge),   "BaseTokenBridge/other-bridge-mismatch");
-        require(l1Bridge.messenger()                   == L1_MESSENGER,        "BaseTokenBridge/l1-bridge-messenger-mismatch");
-        require(l1GovRelay.l2GovernanceRelay()         == address(l2GovRelay), "BaseGovRelay/l2-gov-relay-mismatch");
-        require(l1GovRelay.messenger()                 == L1_MESSENGER,        "BaseGovRelay/l1-gov-relay-messenger-mismatch");
+    function testSparkLendD3MBuffer() public {
+        DirectSparkDaiPlanLike DIRECT_SPARK_DAI_PLAN = DirectSparkDaiPlanLike(addr.addr('DIRECT_SPARK_DAI_PLAN'));
 
+        assertEq(DIRECT_SPARK_DAI_PLAN.buffer(), 50 * MILLION * WAD, "spark-lend-d3m-buffer/invalid-buffer-amount");
 
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done(), "TestError/spell-not-done");
 
-
-        require(l1Bridge.escrow() == L1_ESCROW, "BaseTokenBridge/escrow-does-not-match");
-
-        address[] memory tokens = new address[](2);
-        address[] memory l2tokens = new address[](2);
-        uint256[] memory maxWithdrawals = new uint256[](2);
-
-        tokens[0] = address(usds);
-        tokens[1] = SUSDS;
-
-        l2tokens[0] = L2_USDS;
-        l2tokens[1] = L2_SUSDS;
-
-        maxWithdrawals[0] = type(uint256).max;
-        maxWithdrawals[1] = type(uint256).max;
-
-        _testBaseTokenBridgeIntegration(BaseTokenBridgeParams({
-            l2Bridge:       address(l2Bridge),
-            l1Bridge:       address(l1Bridge),
-            l1Escrow:       L1_ESCROW,
-            tokens:         tokens,
-            l2Tokens:       l2tokens,
-            maxWithdrawals: maxWithdrawals
-        }));
+        assertEq(DIRECT_SPARK_DAI_PLAN.buffer(), 100 * MILLION * WAD, "spark-lend-d3m-buffer/invalid-buffer-amount");
     }
 
-    function testsWhitelistSparkProxyOnLitePsm() public {
-        assertEq(LitePsmLike(MCD_LITE_PSM_USDC_A).bud(ALM_SPARK_PROXY), 0);
+    function testGelatoKeeperTreasuryAddress() public {
+        GelatoPaymentAdapterLike GELATO_PAYMENT_ADAPTER = GelatoPaymentAdapterLike(0x0B5a34D084b6A5ae4361de033d1e6255623b41eD);
+        address GELATO_TREASURY_OLD = 0xbfDC6b9944B7EFdb1e2Bc9D55ae9424a2a55b206;
+        address GELATO_TREASURY_NEW = 0x5041c60C75633F29DEb2AED79cB0A9ed79202415;
```

Comment:
The old address actually [exists in the addresses_wallets.sol](https://github.com/makerdao/spells-mainnet/blob/5a90fdbb5c2f7e1fc2a2531c0549c55e3411f28f/src/test/addresses_wallets.sol#L99). You need to updated it there, otherwise it will become misleading. The old address can be removed as will be no longer in use

Please also use `addr.addr` helper to get both `GELATO_TREASURY` and `GELATO_PAYMENT_ADAPTER` (instead of hardcoding the addresses here)

Reply:
I had to use `wallets.addr` here since the addresses are in the `addresses_wallets.sol` file